lib/widgets/story_view.dart

```diff
@@ -52,6 +52,7 @@ class StoryItem {
   static StoryItem text({
     @required String title,
     @required Color backgroundColor,
+    TextStyle textStyle,
     bool shown = false,
     double fontSize = 18,
```

Comment:
If `textStyle` is introduced, then `fontSize` should probably go away, what do you think?

Reply:
Yeah
Let me fix that